### PR TITLE
LPS-133957 Asset TableReferenceDefinition Tests

### PR DESCRIPTION
**Pull request by ctang3**
https://issues.liferay.com/browse/LPS-133957

### CODE REVIEW

File: modules/apps/asset/asset-list-test/src/testIntegration/java/com/liferay/asset/list/change/tracking/test/AssetListEntryAssetEntryRelTableReferenceDefinitionTest.java [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+		new AggregateTestRule(
+			new LiferayIntegrationTestRule(),
+			PermissionCheckerMethodTestRule.INSTANCE);
+
+	@Before
+	@Override
+	public void setUp() throws Exception {
+		super.setUp();
+
+		_assetEntry = AssetTestUtil.addAssetEntry(group.getGroupId());
```
Comment (Preston-Crary) [inline modules/apps/asset/asset-list-test/src/testIntegration/java/com/liferay/asset/list/change/tracking/test/AssetListEntryAssetEntryRelTableReferenceDefinitionTest.java:57]: If order doesn't matter we should remove the extra spaces between these lines.

File: modules/apps/asset/asset-link-test/src/testIntegration/java/com/liferay/asset/link/change/tracking/test/AssetLinkTableReferenceDefinitionTest.java [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+		new AggregateTestRule(
+			new LiferayIntegrationTestRule(),
+			PermissionCheckerMethodTestRule.INSTANCE);
+
+	@Before
+	@Override
+	public void setUp() throws Exception {
+		super.setUp();
+
+		_assetEntry2 = AssetTestUtil.addAssetEntry(group.getGroupId());
```
Comment (Preston-Crary) [inline modules/apps/asset/asset-link-test/src/testIntegration/java/com/liferay/asset/link/change/tracking/test/AssetLinkTableReferenceDefinitionTest.java:54]: Seems weird we create assetEntry1 before assetEntry2. Let's rename these to productionAssetEntry and publicationAssetEntry.

File: modules/apps/asset/asset-list-test/src/testIntegration/java/com/liferay/asset/list/change/tracking/test/AssetListEntryAssetEntryRelTableReferenceDefinitionTest.java (hunk 1 above)
Comment (Preston-Crary) [reply modules/apps/asset/asset-list-test/src/testIntegration/java/com/liferay/asset/list/change/tracking/test/AssetListEntryAssetEntryRelTableReferenceDefinitionTest.java:57]: Check the other test's setUp() methods too.

File: modules/apps/asset/asset-link-test/src/testIntegration/java/com/liferay/asset/link/change/tracking/test/AssetLinkTableReferenceDefinitionTest.java (hunk 2 above)
Comment (ctang3) [reply modules/apps/asset/asset-link-test/src/testIntegration/java/com/liferay/asset/link/change/tracking/test/AssetLinkTableReferenceDefinitionTest.java]: Updated with requested changes, thanks! Was naming them after the parameter names for `AssetLinkLocalService.addLink` but this is much clearer.

File: modules/apps/asset/asset-list-test/src/testIntegration/java/com/liferay/asset/list/change/tracking/test/AssetListEntryAssetEntryRelTableReferenceDefinitionTest.java (hunk 1 above)
Comment (ctang3) [reply modules/apps/asset/asset-list-test/src/testIntegration/java/com/liferay/asset/list/change/tracking/test/AssetListEntryAssetEntryRelTableReferenceDefinitionTest.java:57]: Updated and removed setUp() extra spaces as much as source formatter permits ✔

**Comment by Preston-Crary** [comment]
ci:forward

**Pull request closed without merging**